packages/components/src/components/Snackbar/Snackbar.stories.tsx

```diff
@@ -34,27 +35,48 @@ export default meta
 
 type Story = StoryObj<SnackbarProps>
 
+const isWeb = Platform.OS !== 'web'
```

Comment:
```suggestion
const isNotWeb = Platform.OS !== 'web'
```
or flip it to an `===` and switch the conditionals below. 

Not sure how long it has been like that (I think it's new?), but `Platform.OS` also returns 'windows' and 'macos' now so my initial thought of renaming to isMobile would also not technically be accurate. Guess it would if reworked to check if it equals iOS or Android.

Reply:
Or better yet: delete the variable since now it's not needed for the offset story and only check it on the Default story.

```suggestion
```